requirements/base.txt

```diff
@@ -0,0 +1,7 @@
+pip==8.1.2
+bumpversion==0.5.3
+wheel==0.29.0
+watchdog==0.8.3
+django==1.9.7
+six
+django-mptt==0.8.6
```

Comment:
Let's think about how to handle this -- we may not want to make it a base requirement (just for testing), and then have some conditional import logic in morango that is OK if django-mptt doesn't exist. We don't want to force it on a project that won't use it, and we don't want to have version conflicts for projects (like Kolibri) that do use it.